tgui/packages/tgui/interfaces/IntegratedCircuit/index.js

```diff
@@ -144,12 +179,17 @@ export class IntegratedCircuit extends Component {
   }
 
   handlePortRelease(event) {
+    window.removeEventListener('mouseup', this.handlePortRelease);
+
+    if (this.timeUntilPortReleaseWorks > performance.now()) {
```

Comment:
I don't understand this in general, seems like it's a very buggy mess if you have to involve nanosecond precision timers to make shit work.

Reply:
This is so that if you are dragging a port, it will no longer drag when you raise your mouse. This is so that you can choose between drag'n'dropping to connect ports and simply clicking to connect ports.

Reply:
stylemistake told me I could resolve this